python/setup.cfg

```diff
@@ -0,0 +1,47 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[metadata]
+name = py-iceberg
```

Comment:
Would anyone be opposed to using a name without dashes, for example `pyiceberg`? Dashes create slight variance that can make a new user's experience a bit clunky. For example after installing it using `pip3 install py-iceberg` they then have to import it as `import py_iceberg`. Also when someone sees some sample code and want to jump right in, they may intuitively try `pip3 install py_iceberg` eventually finding out it should be `pip3 install py-iceberg`.

Reply:
I thought we will do `import iceberg`, not `import py_iceberg`, and the package name is not related to the actual module name. Which one is expected?

Reply:
No they're technically not related but a quick check of the top 10 popular python libraries and all except scikit-learn have identically matching package and module names. `iceberg` feels like a great choice for both places but that would definitely lead to collisions with [this](https://pypi.org/project/iceberg/#description) geoscience library that's reserved the `iceberg` namespace in pypi. Maybe it's not worth it to over-index on the quick-starter who sees `import iceberg` somewhere and immediately tries `pip install iceberg` and accidentally grabs the wrong library. (Sorry I missed the recent syncs on this topic, looking forward to joining the future ones!)

Reply:
@samredai  yep, the name was discussed in the recent two community sync meetings and we finally voted to use this name. 

As PEP8 mentioned
```
Python packages should also have short, all-lowercase names, although the use of underscores is discouraged.
```
So it seems `py-iceberg` is more clear because users cannot use `import py-iceberg` and will then use `import iceberg`.